### PR TITLE
[Component Governance] Explicit dependency Portable.BouncyCastle added.

### PR DESCRIPTION
**Pull request by dannyjdev**
Build verified that this version removes the alert.

### CODE REVIEW

File: tests/Validation.PackageSigning.Core.Tests/Validation.PackageSigning.Core.Tests.csproj [hunk 1]
```diff
@@ -71,6 +71,10 @@
     <PackageReference Include="Newtonsoft.Json">
       <Version>12.0.2</Version>
     </PackageReference>
+    <!-- Explicit dependency added for component governance; Root dependency is Test.Utility -->
```
Comment (agr) [inline tests/Validation.PackageSigning.Core.Tests/Validation.PackageSigning.Core.Tests.csproj:74]: What is `Test.Utility`? Can we update the reference there instead?

Comment (agr) [inline tests/Validation.PackageSigning.Core.Tests/Validation.PackageSigning.Core.Tests.csproj:74]: Is it client's [Test.Utility](https://github.com/NuGet/NuGet.Client/tree/dev/test/TestUtilities/Test.Utility)? Shouldn't it be fixed there then?

Comment (agr) [reply tests/Validation.PackageSigning.Core.Tests/Validation.PackageSigning.Core.Tests.csproj:74]: It seems they are already on 1.8.10 there. Should we just update our version?

**Review by agr — APPROVED** [review]
Please see if we can simply reference newer `Test.Utility` that would pull an updated dependency.

**Comment by dannyjdev** [comment]
> Please see if we can simply reference newer `Test.Utility` that would pull an updated dependency.

Updated only the test.utility version, and changed tests since error logs changed on the following PR https://github.com/NuGet/NuGet.Client/pull/3992/files.

**Pull request merged**